### ISSUE TITLE
Doku options types

### ISSUE THREAD

**Issue by ximex** [mapbox/shp-write#28]
look at the `options.types.line`. It should be: `options.types.polyline`

current:
```js
var options = {
    folder: 'myshapes',
    types: {
        point: 'mypoints',
        polygon: 'mypolygons',
        line: 'mylines'
    }
}
```
correct:
```js
var options = {
    folder: 'myshapes',
    types: {
        point: 'mypoints',
        polygon: 'mypolygons',
        polyline: 'mylines'
    }
}
```

**Comment by tmcw** [comment]
Have you tested this to be true? I haven't looked at this module in a while, but it really appears like [the option would be called line](https://github.com/mapbox/shp-write/blob/a0cf93d0ad72ef038e41cbbcf823ea5aabde2cd3/src/geojson.js#L1-L4).

**Comment by ximex** [comment]
look at:
- https://github.com/mapbox/shp-write/blob/master/src/zip.js#L22
- https://github.com/mapbox/shp-write/blob/master/src/geojson.js#L2

`module.exports.line = justType('LineString', 'POLYLINE');` this "POLYLINE" is compared with the "line/polyline". Maybe change all "POLYLINE" and "line" to "LineString" as it is named in geojson

**Comment by tmcw** [comment]
Yes, but the code, as it currently stands, would work correctly with the example in the readme, and not with the suggested example, right?

**Comment by ximex** [comment]
`options.types[l.type.toLowerCase()]`

`l.type.toLowerCase()` = `polyline`
`options.types` = `{line: "mypolylines"}`
so:
`options.types["polyline"]` = `undefined`
but with:
`options.types` = `{polyline: "mypolylines"}`
`options.types["polyline"]` = `mypolyline`

the example in the readme didn't work for me. i have changed it that it works.
